From 83b703c6b82113b9c98fb10447bcda275bb4d5da Mon Sep 17 00:00:00 2001
From: "Bobby R. Bruce" <bbruce@ucdavis.edu>
Date: Mon, 10 Jan 2022 11:28:05 -0800
Subject: [PATCH 109/757] stdlib: Remove stdlib README.md

This README.md is outdated and incompleted. User's wishing to learn
about the gem5 stdlib should reference the gem5 website:
https://www.gem5.org/documentation/gem5-stdlib/overview

Issue-on: https://gem5.atlassian.net/browse/GEM5-1019
Change-Id: Ib66bd748d517708833ac591515601f206ce4728a
Reviewed-on: https://gem5-review.googlesource.com/c/public/gem5/+/55323
Reviewed-by: Jason Lowe-Power <power.jg@gmail.com>
Maintainer: Jason Lowe-Power <power.jg@gmail.com>
Tested-by: kokoro <noreply+kokoro@google.com>
---
 src/python/gem5/README.md | 37 -------------------------------------
 1 file changed, 37 deletions(-)
 delete mode 100644 src/python/gem5/README.md

diff --git a/src/python/gem5/README.md b/src/python/gem5/README.md
deleted file mode 100644
index 344034d..0000000
--- a/src/python/gem5/README.md
+++ /dev/null
@@ -1,37 +0,0 @@
-# The gem5 Components Library
-
-**IMPORTANT NOTE:** This is a Work-In-Process Documentation. This will be expanded and completed in later revisions of the components library.
-
-This is a high-level overview of what this library is.
-
-## Philosophy
-
-Like the [Zen of Python](https://www.python.org/dev/peps/pep-0020/), the gem5 Components Library has a set of guiding principles.
-Note, these are note rules, and they are meant to be *bent* if needed (but maybe not broken).
-
-### Components are extensible, not configurable
-
-We prefer *extensibility* instead of *configurability*.
-Instead of each component taking many different parameters, we have decided to make many different components.
-For instance, instead of having one core component which takes a parameter of the type (e.g., in-order or out-of-order), we specify multiple different components, an `InOrderCPU` and an `OutOfOrder` CPU.
-
-### Components use easy to remember names
-
-We prefer longer and easier to remember names than shorter or jargon names.
-
-## Structure of the components library
-
-### Boards
-
-### Processors
-
-### Memories
-
-### Cache hierarchies
-
-## Contributing to the components library
-
-### Code style
-
-- Use [Black](https://black.readthedocs.io/en/stable/) to format your code.
-- Docstring should follow the [ReST style and Sphinx](https://www.sphinx-doc.org/)
-- 
1.8.3.1

